src/include/kdump/dialogs.rb

```diff
@@ -539,37 +544,44 @@ def kdump_memory_widget
             HStretch()
           )
         ),
+        VSpacing(1),
         Left(
           HBox(
             Left(Label(_("Usable Memory [MiB]:"))),
             Left(Label(Id("usable_memory"), "0123456789")),
             HStretch()
           )
         ),
-        Left(
-          IntField(
-            Id("allocated_low_memory"),
-            Opt(:notify),
-            low_label,
-            min,
-            max,
-            0
-          )
-        )
+        VSpacing(1),
+        Left(IntField(Id("allocated_low_memory"),
+          Opt(:notify),
+          low_label,
+          low_min,
+          low_max,
+          0)),
+        Left(Label(low_range))
       ]
       if Kdump.high_memory_supported?
-        min = Kdump.memory_limits[:min_high].to_i
-        max = Kdump.memory_limits[:max_high].to_i
+        high_min = Kdump.memory_limits[:min_high].to_i
+        high_max = Kdump.memory_limits[:max_high].to_i
+        high_default = Kdump.memory_limits[:default_high].to_i
+        # TRANSLATORS: Each momory value is based on MiB
+        high_range = format(_("(%{high_min} - %{high_max}, default: %{high_default})"),
```

Comment:
do we really need new translatable string for it? Why not use `_("(%{min} - %{max}, default: %{default})"` and have it consistently translated?